src/librustc_typeck/diagnostics.rs

```diff
@@ -4698,6 +4698,52 @@ element type `T`. Also note that the error is conservatively reported even when
 the alignment of the zero-sized type is less than or equal to the data field's
 alignment.
 "##,
+
+
+E0908: r##"
+A method was called on a raw pointer whose inner type wasn't completely known.
+
+For example, you may have done something like:
+
+```
```

Comment:
This makes me wonder if it is possible to tell the doc comment which epoch it should use.

Reply:
not really. there were plans for making the extended messages dynamic, but they didn't really take off.

Reply:
@Manishearth I mean something like

``````markdown

```rust,epoch2018
let foo: Box<dyn Debug> = box 5;
```

```rust,epoch2015
let foo: Box<Debug> = Box::new(5);
```

``````

Reply:
Hmmm.. Would be nice to have. Out of scope for this PR though 😄 

Anyway, this specific warning only occurs on Rust 2018+, and the extended error stays the same.